CONTRIBUTING.md

```diff
@@ -41,11 +41,10 @@ accept your pull requests.
 
 1.  [Prepare your environment for Node.js setup][setup].
 
-1.  Install dependencies:
+1.  Install dependencies by following the instructions in
```

Comment:
It might be worth splitting this into two parts, there are the sample tests, run via `npm run sample-tests`, which require that someone:

```
cd samples
npm link ../
npm i
```

There are also the unit tests, which do require that someone run `npm i` in the root directory, and are run via `npm t`.

Reply:
Is this what you meant or would you rather see 2 & 3 combined into one section that describes the three types of tests with installation commands?